src/Studio/Plugins/Spectrogram2D/Spectrogram2DPlugin.cpp

```diff
@@ -62,18 +62,11 @@ void Spectrogram2DPlugin::RegisterAttributes()
 
 	// multi view
 	AttributeSettings* attributeSettings = RegisterAttribute("Multi View", "multiView", "", ATTRIBUTE_INTERFACETYPE_CHECKBOX);
-#ifdef NEUROMORE_BRANDING_ANT
```

Comment:
Please don't touch unrelated PR content.

Reply:
@dioptre please checkout to the latest commit I have updated my code with the latest version.